### DISCUSSION TITLE
Reverse playlist order

### DISCUSSION THREAD

**Discussion by derivativeoflog7** [ReVanced/revanced-discussions#272] [Question]
I know this request sounds weird at first, but I've found a lot of playlists for series of videos which are in the wrong order (the last episode is played first and it goes backwards to the first), so I think a feature to have the app play a playlist in reverse would help with this

**Comment by thebestnom** [comment]
YouTube doesn't allow to order the playlist as it actually sends an order command to the server 😅 (just tested on pc) so you arent allowed to change the playlist as you didn't created it (best I can think of is copping the playlist first to you

**Reply by AnnanFay** [reply]
> (best I can think of is copping the playlist first to you

Is it possible to reverse a playlist after copying it? 

I tried copying a large playlist as a work-around, but it didn't seem like there is a feature there.

Maybe an intermediate feature could simply be a button to reverse your own playlist and ones you have copied. (Or I'm missing where this already exists)

**Comment by thebestnom** [comment]
Also continued searchng, found out they disabled copying a whole playlist on youtube (but they do allow it on youtube music) which means copying a playlist is a great addition to vanced to allow what you asking for

**Comment by derivativeoflog7** [comment]
> YouTube doesn't allow to order the playlist as it actually sends an order command to the server sweat_smile (just tested on pc) so you arent allowed to change the playlist as you didn't created it (best I can think of is copping the playlist first to you

Simply having the client fetch a playlist and play it in reverse order is definitely doable - there already [browser extensions](https://chrome.google.com/webstore/detail/improve-youtube-video-you/bnomihfieiccainjcjblhegjgglakjdd/related) to do this

**Comment by royaltongue** [comment]
I know this is an old thread, but I'd love this feature too. So many let's-play playlists are backwards and it makes it impossible to watch

**Comment by sneezeman** [comment]
Plenty of podcasts, too. It's surprisingly difficult to find a solution for this on mobile.
